internal/corazawaf/waf.go

```diff
@@ -119,6 +119,9 @@ type WAF struct {
 	// Array of logging parts to be used
 	AuditLogParts types.AuditLogParts
 
+	// Audit log format
+	AuditLogFormat types.AuditLogFormat
```

Comment:
Audit log is pluggable hence users might define their own, I don't think we can support a strict type for this.